packages/lodestar-cli/src/cmds/beacon/cmds/init/index.ts

```diff
@@ -14,7 +14,13 @@ export function builder(yargs: Argv<IBeaconArgs>): Argv<IBeaconArgs> {
   yargs.options(beaconInitOptions);
   const args = parseArgs(yargs) as IBeaconInitArgs;
   if (args.templateConfigFile) {
-    yargs.config(readBeaconConfig(args.templateConfigFile));
+    try {
+      yargs.config(readBeaconConfig(args.templateConfigFile));
+    } catch(error) {
+      // eslint-disable-next-line no-console
+      console.log(` ✖ ${error.message}\n`);
+      process.exit(1);
+    }
```

Comment:
Errors thrown here are not catched by yargs.fail handler? I would avoid using process.exit() anywhere but a very predictable spot, like in the root index of the CLI.

Reply:
yeah i wasn't able to figure out how get the errors to be thrown to the top-level.  for now, i'll take out that process.exit, but after this PR, i might start to look into refactoring mergeBeaconOptions and see if maybe that's the culprit